src/game/laramisc.c

```diff
@@ -31,6 +31,12 @@ void LaraControl(int16_t item_num)
     ROOM_INFO *r = &g_RoomInfo[item->room_number];
     int32_t room_submerged = r->flags & RF_UNDERWATER;
 
+    if (g_Lara.interact_target.is_moving
+        && g_Lara.interact_target.move_count++ > 90) {
+        g_Lara.interact_target.is_moving = 0;
+        g_Lara.gun_status = LGS_ARMLESS;
+    }
```

Comment:
What do we use this guard, `move_count`, for? Do we ever hit this scenario?

Reply:
This is a check to make sure Lara doesn't get indefinitely stuck if for some reason if she fails to pick up an item. 90 must have been some threshold Core chose.